source/plugins/music/README.md

```diff
@@ -67,6 +67,22 @@ This mode is not supported for now.
 
 </details>
 
+<details>
+<summary>YouTube Music</summary>
+
+Extract the *playlist* URL of the playlist you want to share.
+
+To do so, Open YouTube Music and select the playlist you want to share.
+
+Extract the source link from copying it from the address bar:
+```html
```

Comment:
why is this code block highlighted as `html`? surely it doesn't need a language highlight

Reply:
agreed, removed.